src/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Composer;
+
+use Composer\Script\Event;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+
+class ScriptHandler
+{
+    protected static $destination = '/src/Bridge/Symfony/Bundle/Resources/public/swagger-ui/dist';
```

Comment:
Not sure to follow here. It was in `public` so I put it back in `public` so that it works seamlessly. 

From what I understand the `assets` command copies content from `public` to `web`. So, you want files to be copied to `web/`? 

What are you calling standard edition? The `api-platform/api-platform` package? `composer create-package` will run `composer install` and will run this if we have the same `scripts` lines in it's `composer.json`.

> when installing the bundle in an existing Symfony project

How should this work? A composer event? Related to an existing symfony command? 

Please give me more informations about your needs:
1. Where should the `dist` directory content be copied to?
2. How is it supposed to be triggered (Symfony Command, Composer Event etc.)